package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@filecoin-station/simple-subnet-api",
```

We usually use npm package names that match the GitHub repository name.

```suggestion
  "name": "@checkernetwork/simple-subnet-api",
```

See https://www.npmjs.com/search?q=checkernetwork

---

Great, I wasn't aware that we have updated npm registry name too!